docs/development/releasing.md

```diff
@@ -320,17 +321,24 @@ Then manually create distributions for each platform and upload them:
 
 ```sh
 # Checkout the version to re-upload.
-git checkout vTHE.RELEASE.VERSION
+git checkout vX.Y.Z
 
-# Do release build, specifying one target architecture.
-./script/bootstrap.py --target_arch [arm|x64|ia32]
-./script/build.py -c R
-./script/create-dist.py
+# Create release build
+gn gen out/Release --args="import(\"//electron/build/args/release.gn\") $GN_EXTRA_ARGS"
 
-# Explicitly allow overwritting a published release.
+# To compile for specific arch, instead set
+gn gen out/Release-<TARGET_ARCH> --args='... target_cpu = "[arm|x64|ia32]"'
```

Comment:
I think this would be clearer as: 
```gn gen out/Release-<TARGET_ARCH> --args='import(\"//electron/build/args/release.gn\") target_cpu = "[arm|x64|ia32]"''```